pkg/allocateipip/allocate_ipip_addr.go

```diff
@@ -155,11 +155,15 @@ func removeHostTunnelAddr(ctx context.Context, c client.Interface, nodename stri
 // with some space. Stores the result in the host's config as its tunnel
 // address.
 func assignHostTunnelAddr(ctx context.Context, c client.Interface, nodename string, ipipCidrs []net.IPNet) {
+	attrs := map[string]string{
+		ipam.AttributeNode: nodename,
+		ipam.AttributeType: ipam.AttributeTypeIPIP,
+	}
 	args := ipam.AutoAssignArgs{
 		Num4:      1,
 		Num6:      0,
-		HandleID:  nil,
-		Attrs:     nil,
+		HandleID:  &nodename,
```

Comment:
Probably a better handle than just the nodename, but couldn't think of one in a few seconds.

Reply:
Suggest sticking some prefix on it.  Might come in useful later; will certainly make it easier to see what that handle was for in a dump.  `ipip-tunnel-`?